mev-rs/src/blinded_block_relayer/api/server.rs

```diff
@@ -24,10 +24,11 @@ async fn handle_get_proposal_schedule<R: BlindedBlockRelayer>(
 
 async fn handle_submit_bid<R: BlindedBlockRelayer>(
     State(relayer): State<R>,
+    Query(with_cancellations): Query<bool>,
```

Comment:
following the spec, this will fail

I think you can try `Query<usize>` but could also just do `Query<String>` and something like

```rust
let with_cancellations = if with_cancellations == "1" { true } else { false };
```